homeassistant/components/zwave_js/websocket_api.py

```diff
@@ -0,0 +1,51 @@
+"""Websocket API for Z-Wave JS."""
+
+from dataclasses import asdict
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.components.websocket_api.connection import ActiveConnection
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DATA_CLIENT, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+ID = "id"
+ENTRY_ID = "entry_id"
+TYPE = "type"
+
+
+@callback
+def async_register_api(hass: HomeAssistant) -> None:
+    """Register all of our api endpoints."""
+    websocket_api.async_register_command(hass, websocket_network_status)
+
+
+# Decorator causes typing error from mypy, ignoring locally until it is
+# fixed in the core websocket_api class
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {vol.Required(TYPE): "zwave_js/network_status", vol.Required(ENTRY_ID): str}
+)
+async def websocket_network_status(
```

Comment:
Actually, this is not async. There is no await
```suggestion
@websocket_api.websocket_command(
    {vol.Required(TYPE): "zwave_js/network_status", vol.Required(ENTRY_ID): str}
)
@callback
def websocket_network_status(
```